broadcast.php

```diff
@@ -15,11 +15,11 @@
 include 'header.php';
 
 // URL parameters
-$action = $_REQUEST['action'];
-$text = trim($_REQUEST['text']);
-$request_response = trim($_REQUEST['response']);
-$confirmed = $_REQUEST['confirm'];
-$communications_id = $_REQUEST['id'];
+$action = isset($_REQUEST['action']) ? $_REQUEST['action'] : '';
```

Comment:
See: http://stackoverflow.com/questions/4261133/php-notice-undefined-variable-notice-undefined-index-and-notice-undef/4261200#4261200

> Relying on the default value of an uninitialized variable is problematic in the case of including one file into another which uses the same variable name. It is also a major security risk with register_globals turned on. E_NOTICE level error is issued in case of working with uninitialized variables, however not in the case of appending elements to the uninitialized array. isset() language construct can be used to detect if a variable has been already initialized. Additionally and more ideal is the solution of empty() since it does not generate a warning or error message if the variable is not initialized.

To be honest, I like eliminating PHP notice-level warning messages instead of just turning them off and ignoring them, and doing this accomplishes that.